packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/troubleshooting.doc.ts

```diff
@@ -0,0 +1,105 @@
+import type {LandingTemplateSchema} from '@shopify/generate-docs';
```

Comment:
Pulled the rug on you a bit here, sorry. I moved these pages to staticPages/pages recently. Would you move this into the new folder?

![image](https://github.com/Shopify/ui-extensions/assets/29490857/858e4e14-e6d8-4347-a11d-d6894c7f1d7f)